banzai/images.py

```diff
@@ -172,6 +172,8 @@ def read_images(image_list, pipeline_context):
     for filename in image_list:
         try:
             image = Image(pipeline_context, filename=filename)
```

Comment:
Can probably remove `if self.site is not None and self.instrument is not None:` from line 45 of `images.py`

Reply:
I think we want that there so that it does not search the database if instrument or site are None. If so, a search through the db will raise and exception and any instance of Image where the site and instrument do not correspond to a telescope will fail to initilize

Reply:
I agree. turnerm further pointed out that the entire block can be removed, i.e. 45, 48 and 49, now that get_telescope_id returns None now.